rma/models/rma.py

```diff
@@ -1134,24 +1134,28 @@ def create_replace(self, scheduled_date, warehouse, product, qty, uom):
         self._ensure_can_be_replaced()
         moves_before = self.delivery_move_ids
         self._action_launch_stock_rule(scheduled_date, warehouse, product, qty, uom)
-        new_move = self.delivery_move_ids - moves_before
-        if new_move:
-            self.reception_move_id.move_dest_ids = [(4, new_move.id)]
-            self.message_post(
-                body=_(
-                    "Replacement: "
-                    'Move <a href="#" data-oe-model="stock.move" '
-                    'data-oe-id="%d">%s</a> (Picking <a href="#" '
-                    'data-oe-model="stock.picking" data-oe-id="%d">%s</a>) '
-                    "has been created."
-                )
-                % (
-                    new_move.id,
-                    new_move.name_get()[0][1],
-                    new_move.picking_id.id,
-                    new_move.picking_id.name,
+        new_moves = self.delivery_move_ids - moves_before
+        if new_moves:
```

Comment:
No need of this extra `if`. Just put the `for` and check if body has content.

Reply:
Simplified